server/src/models/client.model.js

```diff
@@ -0,0 +1,81 @@
+require("dotenv").config();
```

Comment:
More so general comment reminder for updating some fields/adding new ones. I know Daniel has a few comments related to date_added.